esp/src/eclwatch/QuerySetDetailsWidget.js

```diff
@@ -262,6 +263,9 @@ define([
                 this.librariesUsedTab.set("tooltip", tooltip);
             } else if (name === "Clusters") {
                 if (lang.exists("ClusterQueryState.length", newValue)) {
+                    newValue.ClusterQueryState[0].MixedNodeStates ? dom.byId(this.id + "SuspendCluster").src = dojoConfig.getImageURL("mixwarn.png") : dom.byId(this.id + "SuspendCluster").src = dojoConfig.getImageURL("errwarn.png");
```

Comment:
The "Clusters" will always be there as long as the IncludeStateOnClusters is set to true.